framework_tvb/tvb/core/entities/model/model_datatype.py

```diff
@@ -229,6 +229,7 @@ class DataTypeMatrix(DataType):
     array_data_min = Column(Float)
     array_data_max = Column(Float)
     array_data_mean = Column(Float)
+    has_volume_mapping = Column(Boolean, nullable=False, default=False)
```

Comment:
I am not convinced this flag should be here (so generic ....).
All inheriting from `BaseVolumeVisualizerForm` should be checked for usage as part of this task, but DatatypeMatrix is inherited and used in many more places.

Reply:
I've put this here because it is needed for MappedArrayVolumeVisualizer/VolumeVisualizerForm, but it is true it's a bit to generic